catalog/USGS/NLCD_RELEASES/USGS_RCMAP_RELEASES_V5_COMPONENTS.jsonnet

```diff
@@ -0,0 +1,358 @@
+local id = 'USGS/NLCD_RELEASES/USGS_RCMAP_RELEASES_V5_COMPONENTS';
```

Comment:
please add an entry for this file to catalog.json in the same directory

Reply:
I believe I did this. Added a line (line 24) in the catalog.jsonnet
ee.link.child_collection('USGS_NLCD_RELEASES_2021_REL_RCMAP_V5_COVER', base_url),